tools/pythonpkg/tests/conftest.py

```diff
@@ -1,33 +1,34 @@
 import os
 import pytest
 import shutil
-from os.path import abspath
+from os.path import abspath, join, dirname, normpath
 import glob
 import duckdb
 
+
 @pytest.fixture(scope="function")
 def duckdb_empty_cursor(request):
     connection = duckdb.connect('')
     cursor = connection.cursor()
     return cursor
- 
+
 
 @pytest.fixture(scope="function")
 def require():
     def _require(extension_name, db_name=''):
         # Paths to search for extensions
+
+        build = normpath(join(dirname(__file__), "../../../build/"))
```

Comment:
Be careful about changing this, I have done this in the past and found out the hard way that the paths aren't the same in the CI.
I also added something similar in a local change yesterday, but replacing all the others seems scary

Reply:
Then it's probably fine, happy to remove this gunk if nothing breaks :thumbsup: